wiki/Debugging.md

```diff
@@ -20,6 +20,62 @@
     while on bootsplash
 
 ### Stuck at bootanimation/splash (or need regular logs but ADB is dead)
+1) Reboot into recovery.
+2) Mount `system` in recovery.
+3) Connect to PC.
+4) Run:
+
+    ```adb shell mount -o rw,remount /system_root```
+
+    ```adb pull /system_root/system/build.prop```

```

Comment:
Why not use one blockquote here?

Reply:
what do you mean by blockquotes ??

>```adb shell mount -o rw,remount /system_root```

>```adb pull /system_root/system/build.prop```

like this ?

Reply:
ok